src/mod/server.mod/servmsg.c

```diff
@@ -1279,6 +1280,28 @@ static void server_resolve_failure(int servidx)
   lostdcc(servidx);
 }
 
+static void do_oident() {
+  char *home = getenv("HOME");
+  char path[121], buf[(sizeof "global{reply \"\"}") + 10];
```

Comment:
I don't know if eggdrop limits the setting itself to 10, but I am sure there are ircds that support more than that. can we just put the whole string into it and let oidentd or the ircd limit if necessary?